content/utils.ts

```diff
@@ -1,9 +1,11 @@
 import path from "node:path";
 import childProcess from "node:child_process";
+
 import LRU from "lru-cache";
+import prettier from "prettier";
```

Comment:
`prettier` is only a devDependency, so if someone uses `@mdn/yari`, but not `prettier`, this will cause the build etc to fail (which actually happened before mdn/content added `prettier`). Therefore, can we continue to `require("prettier")` inline?

Reply:
Ah, gotcha -- in that case, I've added a try-catch clause at the top with a dynamic import statement instead, so that if Prettier is not present, it won't cause everything to stop working!